api/bootstrap/kubeadm/v1beta2/kubeadm_types.go

```diff
@@ -285,8 +299,9 @@ type NodeRegistrationOptions struct {
 	// it will be defaulted to []v1.Taint{'node-role.kubernetes.io/master=""'}. If you don't want to taint your control-plane node, set this field to an
 	// empty slice, i.e. `taints: []` in the YAML file. This field is solely used for Node registration.
 	// +optional
+	// +kubebuilder:validation:MinItems=0
```

Comment:
Hmm, so does this not work then? We still need the exclude?

Reply:
I'll move this into the other thread to dedup